prov/efa/src/rxr/rxr_pkt_type.h

```diff
@@ -390,6 +394,35 @@ static inline struct rxr_atomrsp_hdr *rxr_get_atomrsp_hdr(void *pkt)
 	return (struct rxr_atomrsp_hdr *)pkt;
 }
 
+/* receipt packet headers */
+struct rxr_receipt_hdr {
+	uint8_t type;
+	uint8_t version;
+	uint16_t flags;
+	/* end of rxr_base_hdr */
+	uint32_t tx_id;
+	uint32_t msg_id;
+	uint16_t ret_code;
```

Comment:
```ret_code``` can be negative so int type is more appropriate. Also I will make it ```int32_t```, so the pad can be ```int32_t```.